c10/core/TensorImpl.h

```diff
@@ -1578,7 +1578,11 @@ struct C10_API TensorImpl : public c10::intrusive_ptr_target {
         storage_initialized(),
         "The tensor has a non-zero number of elements, but its data is not allocated yet. "
         "Caffe2 uses a lazy allocation, so you will need to call "
-        "mutable_data() or raw_mutable_data() to actually allocate memory.");
+        "mutable_data() or raw_mutable_data() to actually allocate memory.\n"
```

Comment:
Can we move the caffe2 part of this error message down and mention "if you're using caffe2 ..."?